setup.py

```diff
@@ -154,13 +154,13 @@ def run(self):
 install_requires = [
     "appdirs>=1.4.3,<=1.4.4 ",
     "attrs>=19.3.0,<=20.2.0",
-    "calamus>=0.3.6,<0.3.7",
+    "calamus>=0.3.7,<0.3.8",
     "click-completion>=0.5.0,<=0.5.3",
     "click-plugins==1.1.1",
     "click>=7.0,<=7.1.2",
     "cryptography>=3.4.1,<3.5",
     "cwlgen>=0.4.0,<=0.4.2",
-    "cwltool>=3.0.20200724003302,<=3.0.20200807132242",
+    "cwltool==3.0.20210124104916",
```

Comment:
it makes sense to have requirements like `something>=3.0.1,<=3.0.1` or `something>=3.0.1,<3.1`, even if it's weird to pin a single version like this, but over time, dependabot will change those to dependency ranges that make sense, rather than just updating the number (if you have `something==3.0.1`, dependabot will just change it to e.g. `something==3.0.2`, whereas `something>=3.0.1,<=3.0.1` would get updated to `something>=3.0.1,<=3.0.2`). Same below with rdflib. 

I think in general, we always want to have an upper bound to prevent running into issue where a new version breaks things, but to only constrain it down to the minor version. e.g. `rdflib>=5.0.0,<5.1` for rdflib. That reasonably safe but still allows for patch releases.

Reply:
Thanks for the explainataion!